pan-domain-auth-play_2-6/src/main/scala/com/gu/pandomainauth/service/OAuth.scala

```diff
@@ -85,7 +85,7 @@ class OAuth(config: OAuthSettings, system: String, redirectUrl: String) {
                   user = User(
                     userInfo.given_name,
                     userInfo.family_name,
-                    jwt.claims.email,
+                    jwt.claims.email.getOrElse(userInfo.email),
```

Comment:
This needs copying into the `pan-domain-auth-play_2-7` as well. Maintaining two copies is rubbish so any suggestions welcome for how to avoid duplication.